app/Http/Controllers/ChirpController.php

```diff
@@ -95,13 +97,17 @@ public function destroy(Chirp $chirp): RedirectResponse
      * display the chirps from only the past week
      */
 
-     public function recent()
+    public function recent(Request $request, $days = 14)
```

Comment:
Remove $days parameter since the days input you are handling is already within the $request parameters.